.travis.yml

```diff
@@ -21,7 +21,7 @@ install:
   - docker build --build-arg BUILD_COMMIT=$TRAVIS_COMMIT --target production -t humanconnection/nitro-web .
   - docker-compose -f docker-compose.yml -f docker-compose.travis.yml up -d
   - git clone https://github.com/Human-Connection/Nitro-Backend.git ../Nitro-Backend
-  - git -C "../Nitro-Backend" checkout $TRAVIS_BRANCH || echo "Branch \`$TRAVIS_BRANCH\` does not exist, falling back to \`master\`"
+  - git -C "../Nitro-Backend" checkout $TRAVIS_PULL_REQUEST_BRANCH || git -C "../Nitro-Backend" checkout $TRAVIS_BRANCH || echo "Branch \`$TRAVIS_PULL_REQUEST_BRANCH\` and \`$TRAVIS_BRANCH\` does not exist, falling back to \`master\`"
```

Comment:
@appinteractive it doesn't work like that. If `$TRAVIS_PULL_REQUEST_BRANCH` is not set, then git will checkout `master` and that's it.

Reply:
`git checkout` will checkout master and exit

Reply:
@appinteractive https://travis-ci.com/Human-Connection/Nitro-Web/builds/97073240#L467 :tada: